### ISSUE TITLE
Koopa module - seg_len parmeter

### ISSUE THREAD

**Issue by ajikmr** [thuml/Time-Series-Library#312]
Hello,

Currently seg_len parameter in Koopa module is set to pred_len. As the result can be improved by keeping seg_len parmeter small, hence user should be allowed to input this value directly. And if there  is no input, then it should be set to pred_len( as default option).

I could reduce test error by making seg_len parameter smaller.

python -u run.py   --task_name long_term_forecast   --is_training 1   --root_path ./dataset/traffic/   --data_path traffic.csv   --model_id traffic_96_48   --model Koopa   --data custom   --features M   --seq_len 96   --pred_len 48   --e_layers 2   --d_layers 1   --factor 3   --enc_in 862   --dec_in 862   --c_out 862   --des 'Exp'   --learning_rate 0.001   --itr 1

seg_len =4
Train Loss: 0.3266994 Vali Loss: 0.5076491 Test Loss: 0.5406327
mse:0.0010769274085760117, mae:0.01711210608482361

seg_len =8
Train Loss: 0.2791565 Vali Loss: 0.4712303 Test Loss: 0.4982919
mse:0.0009631062275730073, mae:0.016374005004763603

seg_len =16
Train Loss: 0.2738074 Vali Loss: 0.4537755 Test Loss: 0.4819387
mse:0.0009252480231225491, mae:0.015504314564168453

seg_len = pred_len =48
Train Loss: 0.3449276 Vali Loss: 0.5153565 Test Loss: 0.5424386
mse:0.0010675793746486306, mae:0.01696086674928665

Thanks.

**Comment by wuhaixu2016** [comment]
Hi, great findings!

Actually, the seq_len is an important task-relative hyperparameter that we should carefully tune. In this repo, we fix this hyperparameter as 96 to ensure a fair comparison.
